### PR TITLE
Misc clarifications/cleanups

### PR DESCRIPTION
**Pull request by bjackman**
- Fix some whitespace (my editor does this automatically and I'm too lazy to change it, if you prefer I ignore it I can stop submitting these fixes every time I touch a file...)
- Fix a typo and clarify a description string
- Improve the error message for a certain type of invalid configuration

### CODE REVIEW

File: wa/framework/configuration/plugin_cache.py [hunk 1]
```diff
@@ -73,10 +73,7 @@ def add_configs(self, plugin_name, values, source):
         if self.is_global_alias(plugin_name):
             self.add_global_alias(plugin_name, values, source)
             return
-        for name, value in values.iteritems():
-            self.add_config(plugin_name, name, value, source)
 
-    def add_config(self, plugin_name, name, value, source):
```
Comment (bjackman) [inline wa/framework/configuration/plugin_cache.py]: BTW this means there's no longer an 'add_config' method - I couldn't see any callers so I judged it was fine..

File: wa/framework/target/descriptor.py [hunk 2]
```diff
@@ -153,10 +153,10 @@ def get_descriptions(self):
               ``True`` would suppress that, ensuring that only the base Target
               interface is initialized.
 
-              You may want to set this if there is a problem with one or more default
-              modules on your platform (e.g. your device is unrooted and cpufreq is
-              not accessible to unprivileged users), or if Target initialization is
-              taking too long for your platform.
+              You may want to set this to false if there is a problem with one
```
Comment (setrofim) [inline wa/framework/target/descriptor.py]: "False" should be capitalized and enclosed in double grave quotes, as referring to a Python literal value.

File: wa/framework/target/descriptor.py [hunk 3]
```diff
@@ -153,10 +153,10 @@ def get_descriptions(self):
...
+              or more default modules on your platform (e.g. your device is
+              unrooted and cpufreq is not accessible to unprivileged users), or
+              if Target initialization is taking too long for your platform.
```
Comment (setrofim) [inline wa/framework/target/descriptor.py]: "Target" should be enclosed in grave quotes, refers to an identifier.

**Comment by bjackman** [comment]
Both comments fixed.

**Comment by setrofim** [comment]
"False" still needs grave accents.0

**Comment by bjackman** [comment]
Done

**Pull request merged**